src/routes/(admin)/tests/test-[type]/[id]/question-selection/QuestionSelectionDialog.svelte

```
@@ -0,0 +1,238 @@
+	import { Button } from '$lib/components/ui/button';
+	import CircleCheck from '@lucide/svelte/icons/circle-check';
+	import { DEFAULT_PAGE_SIZE } from '$lib/constants';
+
+	let { open = $bindable(), questions, questionParams, formData } = $props();
+	let tags = $state<string[]>([]);
+	let states = $state<string[]>([]);
+	let searchTimeout: ReturnType<typeof setTimeout>;
+
+	// let's format questions data to match QuestionForSelection interface
+	const questionData: QuestionForSelection[] = $derived(
+		questions.items?.map((question: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)